firefoxThemes/options/options.js

```diff
@@ -7,6 +7,11 @@ const root = document.querySelector(':root');
 function setCss(chosenTheme) {
   if (!chosenTheme) return;
   const {colors} = chosenTheme.definition;
+  root.style.setProperty('--checkbox-color',colors.accentColor);
+  root.style.setProperty('--foreground-color',colors.foregroundColor);
+  root.style.setProperty('--checkbox-border-color',colors.foregroundColor+70);
+  root.style.setProperty('--header-color',colors.headerColor);
+  root.style.setProperty('--checkmark-color',colors.classNameColor);
```

Comment:
I decided to use the `classNameColor` color definition instead as the other color definitions  made it difficult to see the checkmark in the checkboxes.

Reply:
I like the idea! However I am a bit partial to the checkbox esthetic from the jetbrains plugin. 

![image](https://user-images.githubusercontent.com/15972415/132493036-239a807c-498c-4c9d-9fcd-431e192071fe.png)

What do you think about this?

![Peek 2021-09-08 05-42](https://user-images.githubusercontent.com/15972415/132495862-59289d04-9503-41e5-9af2-3da45a70830e.gif)

I can push the changes up if we both agree.

Reply:
Hmm... I'm more partial to the colored-in checkbox idea myself as it adds more coloration to the options page. Though the jetbrain aesthetic is fine too. If using the jetbrain's aesthetic, to keep things synonymous with the plugin, I recommend staying with the &#10003; over the &#9829; as the checkmark.

Reply:
I'll leave it as is. I might play around with the idea of adopting the :heart: checkbox in other platforms.